lib/citizens_advice_form_builder/elements/collections/select.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module CitizensAdviceFormBuilder
+  module Elements
+    module Collections
+      class Select < Base
+        include ActionView::Helpers::FormOptionsHelper
+
+        def render
+          component = CitizensAdviceComponents::Select.new(
+            select_options: items,
+            name: field_id,
+            label: label,
+            options: {
+              hint: hint,
+              optional: optional,
+              error_message: error_message,
+              value: current_value,
+              additional_attributes: { name: field_name }
```

Comment:
Design system currently doesn't allow overriding the id of the field so we need to use this workaround. We can later contribute to the design system and allow custom id for the select component.